docs/reference/content/upgrading.md

```diff
@@ -7,11 +7,39 @@ title = "Upgrade Considerations"
   pre = "<i class='fa fa-level-up'></i>"
 +++
 
-## Upgrading from 3.x
-
-... TODO ...
-
-## Upgrading from mongo-java-driver-reactivestreams 1.12
+# Upgrading to the 4.0 Driver
```

Comment:
Not sure about the increase in header size here, but I wanted something to group together the three drivers.